src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -293,6 +293,44 @@ public void testGetUser_successful() throws Exception {
         Mockito.verify(mockApiClient).get("https://api.sandbox.hyperwallet.com/rest/v4/users/test-user-token", userResponse.getClass());
     }
 
+    @Test
+    public void getUser_WithVerificationDocumentAndRejectReasons() throws Exception {
+        Hyperwallet client = new Hyperwallet("test-username", "test-password");
+        HyperwalletApiClient mockApiClient = createAndInjectHyperwalletApiClientMock(client);
+        HyperwalletUser hyperwalletUser = getHyperwalletUser();
+        final String url = "https://api.sandbox.hyperwallet.com/rest/v4/users/" + "test-userToken";
+        Mockito.when(mockApiClient.get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class))).thenReturn(hyperwalletUser);
+
+        HyperwalletUser user = client.getUser("test-userToken");
+
+        Mockito.verify(mockApiClient).get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class));
+        assertThat(user.getToken(), is(hyperwalletUser.getToken()));
+        assertThat(user.getDocuments().size(), is(1));
+        assertThat(user.getDocuments().get(0).getStatus(), is("INVALID"));
+        assertThat(user.getDocuments().get(0).getReasons().size(), is(1));
+        assertThat(user.getDocuments().get(0).getReasons().get(0).getName(), is(DocumentVerificationReason.DOCUMENT_EXPIRED));
+        assertThat(user.getDocuments().get(0).getReasons().get(0).getDescription(), is("Document has expired"));
+    }
+
+    private HyperwalletUser getHyperwalletUser() {
+        HyperwalletUser user = new HyperwalletUser();
+        user.token("test-userToken")
+                .verificationStatus(VerificationStatus.FAILED);
+        HyperwalletVerificationDocument hyperwalletVerificationDocument = new HyperwalletVerificationDocument();
+        hyperwalletVerificationDocument.setType("test_DRIVERS_LICENSE");
+        hyperwalletVerificationDocument.setCategory("test_IDENTIFICATION");
+        hyperwalletVerificationDocument.setCountry("test_US");
+        Map<String, String> fileList = new HashMap<>();
+        fileList.put("fileName", "fileData");
+        hyperwalletVerificationDocument.setStatus("INVALID");
+        HyperwalletDocumentRejectReason documentRejectReason = new HyperwalletDocumentRejectReason();
+        documentRejectReason.setName(DocumentVerificationReason.DOCUMENT_EXPIRED);
+        documentRejectReason.setDescription("Document has expired");
```

Comment:
we should set description from enum

Reply:
Have removed the description field, as we copy the name and description fields directly from api response, i think it doesn't need to come from enum description, mapping logic between name and description already exists in hw2 code.